misc/Bootcamp_template/appName/appName_labs/English/C/LICENSE

```diff
@@ -0,0 +1,543 @@
+Apache License
```

Comment:
Shall we just put right license here instead of copy-pasting apache.
We can say that by default the license is Apache for all our labs unless specified otherwise